src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -30,13 +31,18 @@ export default function createAdditionalInfoWidget() {
 
       const chevron = qS(el, 'i.fa-angle-down');
       const button = qS(el, 'button');
+      const analyticsEvent = button.dataset.event;
 
       button.addEventListener('click', () => {
         const ariaExpanded = JSON.parse(button.getAttribute('aria-expanded'));
 
         button.setAttribute('aria-expanded', `${!ariaExpanded}`);
         button.parentNode.classList.toggle('form-expanding-group-open');
         chevron.classList.toggle('open');
+
+        if (analyticsEvent) {
+          recordEvent(analyticsEvent);
+        }
```

Comment:
Sooo close!! But I wasn't able to see `analyticsEvent` logged when I added a `console.log` on line 44. 

I think we need a couple final revisions -

1. I think we'll need to access the `dataset` from [`contentContainer`](https://github.com/department-of-veterans-affairs/vagov-content/pull/378/files#diff-ee0b729c9792aae6ad3d58b9060a0542R177) instead of the button. We could probably add that around line 15, where the rest of the variables are initialized.
1. I think `recordEvent` expects an object instead of a string (apologies if I steered you wrong.) I think it should be `recordEvent({ event: analyticsEvent });`. It's just an alias for the standard GA `window.dataLayer.push`.

Reply:
oh dang, yeah you're totally right! that's on me for trying to quickly update rather than verify 🙃will update soon 👍

Reply:
should be good now. I think if we want to add this to the drupal pages, you could just create something like `fieldAlertAnalyticsEvent` on the fieldAlert entity and then just add some additional logic looking for that in the template